pom.xml

```diff
@@ -43,9 +43,19 @@
 			<version>1.47.1</version>
 		</dependency>
 		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			<version>18.0</version>
+		</dependency>
+		<dependency>
 			<groupId>org.elasticsearch</groupId>
 			<artifactId>elasticsearch</artifactId>
-			<version>1.3.2</version>
+			<version>2.3.3</version>
+		</dependency>
+		<dependency>
+			<groupId>com.sun.jna</groupId>
+			<artifactId>jna</artifactId>
+			<version>3.0.9</version>
```

Comment:
- guava_18.0 is necessary to use the es client, see http://stackoverflow.com/questions/33544863/java-elasticsearch-client-always-null
- jna is only needed to supress an ES  warning , not important. Going to remove the lib.